cli/pkg/schema/service.go

```diff
@@ -1,13 +1,35 @@
 package schema
 
+// Supported languages for client generation
+const (
+	LanguageGo     = "go"
+	LanguagePython = "python"
+	LanguageTS     = "ts"
+	LanguageJS     = "js"
+)
+
+// GetSupportedLanguages returns a slice of all supported languages
+func GetSupportedLanguages() []string {
+	return []string{LanguageGo, LanguagePython, LanguageTS, LanguageJS}
+}
+
+// SupportedLanguagesJSONSchema contains the enum definition for JSON schema
+const SupportedLanguagesJSONSchema = "enum=go,enum=python,enum=ts,enum=js"
+
 type ServiceIntent struct {
 	Resource  `json:",inline" yaml:",inline"`
+	Path      string            `json:"path" yaml:"path" jsonschema:"required"`
```

Comment:
If we include this I don't think it should be required configuration, `.` is a reasonable default. I think this config would be exceptional for the majority of brownfields applications (single application in a repo).